include/dlaf/solver/triangular/impl.h

```diff
@@ -505,17 +543,13 @@ void Triangular<backend, device, T>::call_LLT(comm::CommunicatorGrid grid, blas:
     if (this_rank.row() == rank_kk.row()) {
       for (SizeType j_loc = 0; j_loc < distr_b.localNrTiles().cols(); ++j_loc) {
         const LocalTileIndex kj(kk_offset.row(), j_loc);
-        auto tile_diff = hpx::unwrapping([](auto tile_a, const auto& tile_b) {
-          DLAF_ASSERT(equal_size(tile_a, tile_b), tile_a, tile_b);
-          for (auto j = 0; j < tile_a.size().cols(); ++j)
-            blas::axpy(tile_a.size().rows(), T(-1), tile_b.ptr({0, j}), 1, tile_a.ptr({0, j}), 1);
-        });
         // TODO executor
```

Comment:
```suggestion
```
probably?

Reply:
I left this comment here (and elsewhere) because I still have to verify where I have to use NP/HP executors before merging 😉